src/sentry/tasks/post_process.py

```diff
@@ -754,9 +754,15 @@ def process_commits(job: PostProcessJob) -> None:
                     organizations=event.project.organization,
                     provider__in=["github", "gitlab"],
                 )
+                use_fallback = (
+                    features.has(
```

Comment:
can we use has_for_batch here or has_batch no idea which is the right one

Reply:
I think that's meant for a batch check for all projects in an organization, but we're checking the flag at the org level

Reply:
ah okay